frontend/src/components/regions/DueTimeFormModal/Presentation.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import {
+  TextField,
+  Button,
+  Grid,
+  Paper,
+} from "@material-ui/core";
+import Modal from '../../widgets/Modal';
+
+const useStyles = makeStyles({
+
+    dueTimeButton: {
+        marginTop: "0.3rem",
+        marginRight: "0.3rem",
+    },
+});
+
+const DueTimeFormModal = ({ card, closeModal }) => {
+
+  const {dueTime} = card || {};
+  const defaultValue = dueTime ? dueTime.split('').slice(0, 16).join('') : "";
```

Comment:
Can you add a comment here.  It is not clear why you have chosen slice(0,16) instead of slice(0,2345).  The next person to deal with this code will need to waste time looking things up.

Just say something like:
// dueTime is either an empty string or something like this: "put in some example time so that folks can see the format here"